src/models/User.js

```diff
@@ -1,15 +1,19 @@
-const bcrypt = require('bcrypt-nodejs')
-const crypto = require('crypto')
+let bcrypt = require('bcrypt-nodejs')
+let crypto = require('crypto')
+
```

Comment:
That was intentional. The one line of whitespace separates NodeJS package imports from our custom includes. You'll notice that I did that in a lot of other places too.